martha/martha_v3.js

```diff
@@ -70,6 +70,7 @@ const BOND_PROVIDER_FENCE = 'fence';
 const BOND_PROVIDER_ANVIL = 'anvil';
 
 const ACCESS_METHOD_TYPE_NONE = null;
+// eslint-disable-next-line no-unused-vars
 const ACCESS_METHOD_TYPE_GCS = 'gs';
```

Comment:
ToL: One can also comment out the variable?

Reply:
or delete it, whichever my reviewers like best 🙂